CODEOWNERS

```diff
@@ -1,4 +1,11 @@
 * @aiuto @nacl
 
-/deb_packages/  @MarkusTeufelberger @adragomir
-/pkg/rpm/ @nacl
+/deb_packages/  @aiuto
+/docs		@aiuto @nacl @floriographygoth
+
+/pkg/rpm/	@nacl
+/pkg/tests/rpm/	@nacl
+legacy/tests/rpm	@nacl
```

Comment:
So, what's the exact policy on approvers here?  It seems that not having multiple CODEOWNERS could be risky if a change must be reviewed by a CODEOWNER.

Reply:
I think the policy works, in that you can't approve yourself.  But I don't actually know.
But.... once we do it, we can observe the behavior and correct it.

Reply:
I find the only way to figure out some of the github interactions is to iteratively submit and see.